torch/distributed/rpc/api.py

```diff
@@ -24,6 +25,7 @@
     _invoke_rpc_script,
     _set_rpc_timeout,
     _start_rpc_agent,
+    _reset_current_rpc_agent,
```

Comment:
this should be above _set_rpc_timeout?

Reply:
Right. I though formatter could order then, but no.